content/en/docs/chart_template_guide/function_list.md

```diff
@@ -857,18 +857,38 @@ Helm includes the following regular expression functions: [regexFind
 (mustRegexReplaceAllLiteral)](#regexreplaceallliteral-mustregexreplaceallliteral),
 [regexSplit (mustRegexSplit)](#regexsplit-mustregexsplit).
 
-### regexMatch, mustRegexMatch
+### regexMatch
```

Comment:
There are links above and this change breaks that. I don't think you should change the headers or add the `### mustRegexMatch` header here.